### PR TITLE
Code for CrackMapExec Snap package

### PR DESCRIPTION
**Pull request by Xhoenix**
Copied code from https://github.com/JitPatro/CrackMapExec-Snap/blob/main/snap/snapcraft.yaml for submitting to Upstream Repo.

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: crackmapexec
+base: core22
+adopt-info: crackmapexec
+
+license: GPL-3.0-or-later
+source-code: https://github.com/Porchetta-Industries/CrackMapExec
+issues: https://github.com/JitPatro/crackmapexec/issues
```
Comment (mpgn) [inline snap/snapcraft.yaml:7]: Can you put the cme repo ?

**Review by mpgn — CHANGES_REQUESTED** [review]

File: snap/snapcraft.yaml (hunk 1 above)
Comment (GenialHacker) [reply snap/snapcraft.yaml]: Changed it.

**Comment by Xhoenix** [comment]
@mpgn, I changed my username from GenialHacker to Xhoenix

**Comment by Xhoenix** [comment]
Hey @mpgn, how about I add a desktop menu icon to the snap? If you want this, then you can provide me a png file for the desktop menu icon.

**Pull request closed without merging**